sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -708,7 +709,7 @@ object SQLConf {
   val HIVE_VERIFY_PARTITION_PATH = buildConf("spark.sql.hive.verifyPartitionPath")
     .doc("When true, check all the partition paths under the table\'s root directory " +
          "when reading data stored in HDFS. This configuration will be deprecated in the future " +
-         "releases and replaced by spark.files.ignoreMissingFiles.")
+         s"releases and replaced by ${SPARK_IGNORE_MISSING_FILES.key}.")
```

Comment:
@cloud-fan Regarding to your comment https://github.com/apache/spark/pull/19868#issuecomment-571957075, `spark.sql.hive.verifyPartitionPath` can be changed at runtime but `spark.files.ignoreMissingFiles` cannot be. Is it fair replacement?

Reply:
We can just slightly reword that users can use `spark.files.ignoreMissingFiles` instead of saying it's a replacement if you're concerned.  I don't believe this configuration is commonly used enough, and it should be fine.

If users find this is unreasonable, we can un-deprecate it later given the feedback.